controllers/ptpinstance_controller.go

```diff
@@ -207,12 +217,13 @@ func (r *PtpInstanceReconciler) ReconciledDeleted(client *gophercloud.ServiceCli
 
 			r.ReconcilerEventLogger.NormalEvent(instance, common.ResourceDeleted, "PTP instance has been deleted")
 		}
-
-		// Remove the finalizer so the kubernetes delete operation can continue.
-		instance.ObjectMeta.Finalizers = utils.RemoveString(instance.ObjectMeta.Finalizers, PtpInstanceFinalizerName)
-		if err := r.Client.Update(context.Background(), instance); err != nil {
-			return err
-		}
+		/*
```

Comment:
Pls remove all the commented parts in this commit

Reply:
Removed the unwanted